src/components/CardRegistroDoDia.tsx

```diff
@@ -72,22 +73,25 @@ const CardRegistroDoDia = ({ diario, navigation }: Props) => {
                 locale: dateLocale
               })}
             </Caption>
-            <TextButton texto={t('diario.verMais')} onPress={handleVerMais} />
+            <TextButton texto={t('diario.verMais')} onPress={handleVerMais} testID='botaoVerMais'/>
           </View>
           <Categoria
             categoria={Categorias.Sentimentos}
             conteudo={ConteudoSentimentos}
             onPress={handleSentimentos}
+            testID='preencheSentimentos'
           />
           <Categoria
             categoria={Categorias.Habitos}
             conteudo={ConteudoHabitos}
             onPress={handleHabitos}
+            testID='preencheHabitos'
```

Comment:
Seguir padrão de aspas duplas (" ") ao invés aspas simples (' ')  
```suggestion
            testID="preencheHabitos"
```